_data/projects/Maven-Using-CMD.yml

```diff
@@ -0,0 +1,12 @@
+name: Maven-Using-CMD
+desc: Creating a Hello World code project in Java using the Apache Maven build tool and automating it using Jenkins.
+site: https://github.com/NishkarshRaj/Maven-Using-CMD
+tags:
+- Maven
+- Java
+- XML
+- Jenkins
+- Automation
+Up-for-Grabs:
```

Comment:
```suggestion
upforgrabs:
```

Reply:
Thanks alot for the help :smile:

Reply:
@NishkarshRaj you'll have to accept these suggested changes because I don't have permission to make changes to the PR.